media_driver/linux/common/codec/ddi/media_ddi_encode_avc.h

```diff
@@ -180,6 +180,8 @@ class DdiEncodeAvc : public DdiEncodeBase
         DDI_MEDIA_CONTEXT *mediaCtx,
         void              *ptr);
 
+    virtual void ClearPicParams();
```

Comment:
please add 'override' for virtual function which from base class.

Reply:
Added.